.github/workflows/benchmark.yaml

```diff
@@ -0,0 +1,72 @@
+name: benchmark_changes
+on:
+  issue_comment:
+    types: [created]
+
+jobs:  
+  benchmark:
+    name: "Benchmark"
+    if: github.event.issue.pull_request && github.event.comment.body == '/benchmark'
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - uses: izhangzhihao/delete-comment@master
```

Comment:
Ah I can't recall, why are we removing the comments from the bot here?

Reply:
Consider this situation -> I ran the benchmark and got the result on PR, but now I changed something and ran the benchmark again on the PR and got the result on PR. Since the benchmark result is really long the PR can get cluttered easily. And in any case, we only care about the most recent benchmarks, so we aren't losing anything by deleting previous comments from the bot.

Reply:
I don't have any strong opinions about keeping or not keeping the older stats in the comments.

While testing multiple benchmarks on my fork, I got frustrated because of multiple results and a list of `/benchmark` comments. So I added the part to have previous comments by bot deleted.

I am fine with either merging this PR as is and then following up after if we feel that we want all the benchmarks results on PR, or updating this PR to keep all the bot comments and then following up with deleting comments if need be in the future.

How do folks feel about this?

Reply:
Either works for me.